Core/Utilities/DErrno.py

```diff
@@ -176,15 +186,18 @@
               # 113X: Databases
               EDB : "Database Error",
               EMYSQL : "MySQL Error",
-
+              # WMS/Workflow
+              EWMSJDL : "Invalid JDL",
               # DMS/StorageManagement
               EFILESIZE : "Bad file size",
               EGFAL : "Error with the gfal call",
               EBADCKS : "Bad checksum",
               EFCERR : "FileCatalog error",
-
               # RMS
               ERMSUKN : "Unknown RMS error",
+              # TS
+              ETSUNK : "Unknown Transformation System Error",
```

Comment:
The "unknown rms error" is ermsUKN, and here it is etsUNK, would be better to use one abbreviation for unknown.